frontend/src/router/routes.js

```diff
@@ -3,14 +3,19 @@ const routes = [
     path: "/",
     component: () => import("layouts/MainLayout.vue"),
     children: [
-      { path: "/", component: () => import("pages/IndexPage.vue") },
-      { path: "/query", component: () => import("pages/QueryPage.vue") },
+      { path: "", component: () => import("pages/IndexPage.vue") },
```

Comment:
Just a small question: I guess would it not be better to tell "/dashboard" for dashboard page and "/" will anyways should reroute to login page, which would mean www.aora.de/ ---reroute--> www.aora.de/login? and once logged in it will take to www.aora.de/dashboard